test/exercism/daily_test.rb

```diff
@@ -0,0 +1,99 @@
+require_relative '../integration_helper'
+
+class DailyTest < Minitest::Test
+  include DBCleaner
+
+  def test_returns_exercises_for_authorized_language_and_slug
+    fred = User.create(username: 'fred')
+    sarah = User.create(username: 'sarah')
+    ACL.authorize(fred, Problem.new('ruby', 'bob'))
+
+    ex1 = create_exercise_with_submission(sarah, 'ruby', 'bob')
+    Like.create!(submission: ex1.submissions.first, user: sarah)
+    Comment.create!(submission: ex1.submissions.first, user: sarah, body: "I hope that when I die, people say about me, 'Boy, that guy sure owed me a lot of money.'")
+
+    assert_equal [ex1.key], fred.dailies.map(&:key)
```

Comment:
How about adding an unauthorized exercise as well so that this exercise checks that it _only_ returns authorized results?